src/material/tooltip/testing/tooltip-harness.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
+import {TooltipHarnessFilters} from './tooltip-harness-filters';
+
+/** Harness for interacting with a standard mat-tooltip in tests. */
+export class MatTooltipHarness extends ComponentHarness {
+  private _optionalPanel = this.documentRootLocatorFactory().locatorForOptional('.mat-tooltip');
+  static hostSelector = '.mat-tooltip-trigger';
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search
+   * for a tooltip trigger with specific attributes.
+   * @param options Options for narrowing the search.
+   * @return a `HarnessPredicate` configured with the given options.
+   */
+  static with(options: TooltipHarnessFilters = {}): HarnessPredicate<MatTooltipHarness> {
+    return new HarnessPredicate(MatTooltipHarness, options);
+  }
+
+  /** Opens the tooltip. */
+  async open(): Promise<void> {
+    return (await this.host()).hover();
+  }
```

Comment:
bikeshedding: what about "show" and "hide" for tooltip? For whatever reason, it sounds weird to "open" a tooltip. (kind of like you wouldn't say "Get in a train" or "Get on a car")

Reply:
Sure, that aligns with the API on `MatTooltip`.